tests/PhpUnit/JsonSchemaConstraintTest.php

```diff
@@ -53,7 +54,8 @@ public function testValidSchema()
 JSON;
         $response = json_decode($response);
 
-        self::assertTrue($this->constraint->evaluate($response, '', true), $this->constraint->evaluate($response));
```

Comment:
Current behavior is described by PHPUnit https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/Constraint.php#L33-L35

In theory setUp method is creating a new instance of the constraint and of the Validator. If any state is shared then we have to fix that test for make it stateless.